components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+* Use the official Git repository (https://github.com/symfony/phpunit-bridge).
+
+.. include:: /components/require_autoload.rst.inc
+
+Usage
+-----
+
+Once the component installed, it automatically registers a
+`PHPUnit event listener`_. This listener simply registers a `PHP error handler`_
+called `DeprecationErrorHandler`. After running your PHPUnit tests again, you
+will have a similar report:
+
+.. image:: /images/components/phpunit_bridge/report.png
+
+The summary includes:
+
+* **Unsilenced** reports deprecation notices that were triggered without the recommended @-silencing operator;
+* **Legacy** deprecation notices denote tests that explicitly test some legacy interfaces.
+* **Remaining/Other** deprecation notices are all other (non-legacy) notices, grouped by message, test class and method.
+
+Trigger deprecation notices
+---------------------------
+
+Deprecation notices can be triggered by using:
+
+    @trigger_error('Your deprecation message', E_USER_DEPRECATED);
+
+Without the @-silencing operator, users would need to opt-out from deprecation
+notices. Silencing by default swaps this behavior and allows users to opt-in
+when they are ready to cope with them (by adding a custom error handler like the
+one provided by this bridge.) When not silenced, deprecation notices will appear
+in the **Unsilenced** section of the deprecation report.
+
+Mark tests as legacy
+--------------------
+
+There are four ways to mark a test as legacy:
+
+* Make its class start with the `Legacy` prefix
+* Make its method start with `testLegacy`
+* Make its data provider start with `provideLegacy` or `getLegacy`
+* Add the `@group legacy` annotation to its class or method
+
+Configuration
+-------------
+
+In case you need to inspect the stack trace of a particular deprecation
+triggered by your unit tests, you can set the `SYMFONY_DEPRECATIONS_HELPER`
+`environment variable`_ to a regular expression that matches this deprecation's
+message, encapsed between `/`. For example, with:
+
+.. configuration-block::
+
+    .. code-block:: xml
+
+        <?xml version="1.0" encoding="UTF-8"?>
+
+        <!-- http://phpunit.de/manual/4.1/en/appendixes.configuration.html -->
+        <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                 xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.1/phpunit.xsd"
+                 backupGlobals="false"
+                 colors="true"
+                 bootstrap="app/autoload.php"
+        >
+            <php>
+                <ini name="error_reporting" value="-1" />
+            </php>
+
+            <testsuites>
+                <testsuite name="Project Test Suite">
+                    <directory>tests</directory>
+                </testsuite>
+            </testsuites>
+
+            <php>
+                <server name="KERNEL_DIR" value="app/" />
+                <env name="SYMFONY_DEPRECATIONS_HELPER" value="/foobar/" />
```

Comment:
let's only show the PHP tag + env tag. There is no need to show the complete file